hermes-client/src/main/java/pl/allegro/tech/hermes/client/HermesClientBuilder.java

```diff
@@ -44,7 +44,7 @@ public HermesClient build() {
         return hermesClient;
     }
 
-    public HermesClientBuilder withURI(URI uri) {
+    public HermesClientBuilder withUri(URI uri) {
```

Comment:
We need to be very careful with changing the hermes-client API. Usually, we try not to break backward compatibility to make the upgrade process easier for users. I would suggest turning off the check that validates that the method and class names are in the camel case. Or at least exclude the hermes-client module from this validation.